### ISSUE TITLE
Timm model fbnetv3_b bfloat16 training accuracy got failed

### ISSUE THREAD

**Issue by mengfei25** [intel/torch-xpu-ops#1765]
### 🐛 Describe the bug

Observed with latest pytorch (f7a5ad6), fbnetv3_b bfloat16 training got fail_accuracy
last known good: 2625c70aecc6eced1dbe108279feab7509733bef
```
python benchmarks/dynamo/timm_models.py --accuracy --bfloat16 -d xpu -n10 --training  --only fbnetv3_b --cold-start-latency --backend=inductor
```

### Versions

pytorch: f7a5ad6c2987ee5a83aa5d868cee3b8067d3de94
torchvision: 966da7e46f65d6d49df3e31214470a4fe5cc8e66
timm: 5d535d7a2d4b435b1b5c1177fd8f04a12b942b9a

device: PVC 1100
driver: LTS 803.61
kernel: 5.15.0-73-generic
igc version: 1.0.220-803~22.04

**Comment by mengfei25** [comment]
Fixed after rebase FP64 patch

**Comment by mengfei25** [comment]
release 2.8 rc3 is also passed

**Comment by chuanqi129** [comment]
But latest nightly failed on this model

**Comment by mengfei25** [comment]
Passed in https://github.com/intel/torch-xpu-ops/actions/runs/16225051165
